README.md

```diff
@@ -1 +1,141 @@
-# Portifoliowebsite
\ No newline at end of file
+# Portfolio Website
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Kindly remove all placeholder texts that describe what should be done in each section to make your README file looks more professional.